docker/python3-scientific/create_zarr_optimus.py

```diff
@@ -282,18 +283,25 @@ def add_expression_counts(data_group, args):
     exp_counts_group = data_group.zeros('expression',
                                         compressor=COMPRESSOR,
                                         shape=csr_exp_counts.shape,
-                                        chunks=(CHUNK_ROW_SIZE, csr_exp_counts.shape[1]),
+                                        chunks=(CHUNK_ROW_SIZE, CHUNK_COL_SIZE),
                                         dtype=np.float32)
 
     # load a chunks from the expression count matrix and update the corresponding chunk in expression matrix in zar
     for i in range(0, csr_exp_counts.shape[0], CHUNK_ROW_SIZE):
-        # check if it is possible to make a full chunk of data
-        if i + CHUNK_ROW_SIZE <= csr_exp_counts.shape[0]:
-            exp_counts_group[i:i + CHUNK_ROW_SIZE, ] = csr_exp_counts[i:i + CHUNK_ROW_SIZE, ].toarray()
-        else:
-            # not enough data remaining, so make the final set of data less than a chunk
-            j = csr_exp_counts.shape[0] - i
-            exp_counts_group[i:i + j, ] = csr_exp_counts[i:i + j, ].toarray()
+      for j in range(0, csr_exp_counts.shape[1], CHUNK_COL_SIZE):
```

Comment:
Please fix the indentations:
```python
for j in range(0, csr_exp_counts.shape[1], CHUNK_COL_SIZE):
    # check if it is possible to make a full row chunk of data, otherwise adjust to the correct size
    p= CHUNK_ROW_SIZE
    if i + CHUNK_ROW_SIZE > csr_exp_counts.shape[0]:
        p = csr_exp_counts.shape[0] - i

    # check if it is possible to make a full row chunk of data, otherwise adjust to the correct size
    q = CHUNK_COL_SIZE
    if j + CHUNK_COL_SIZE > csr_exp_counts.shape[1]:
        q = csr_exp_counts.shape[1] - j

    #insert the chunk
    exp_counts_group[i:i + p, j:j + q] = csr_exp_counts[i:i + p, j: j + q].toarray()
```